src/frontend/apps/desk/src/pages/teams/TeamLayout.tsx

```diff
@@ -0,0 +1,26 @@
+import { PropsWithChildren } from 'react';
```

Comment:
Should it be placed under `src/features/teams/components`? I might be wrong to please feel free to ignore my comment.

Reply:
I didn't really understood what do you mean by internal and external bu trust you on this one

Reply:
Internal is in `src` folder.